test/valid-packages-test.js

```diff
@@ -306,6 +306,21 @@ packages.map(function(pkg) {
                 pkgName(pkg) + ": Need to remove \"/\" at the front or the last of basePath in package.json");
         }
       }
+  }
+  packageVows[pname + ": There can't be duplicate keywords"] = function(pkg) {
+      var json = parse(pkg, true);
+      var keywords = json.keywords;
+      var dul_key = [];
+      if (keywords != undefined) {
+        keywords.sort();
+        dul_key = keywords.filter (function (dul, i, temp) {
+            if (i >= 0 && dul === temp[i-1]) {
+              return dul;
+            }
+        });
+      }
+      assert.ok(dul_key.length <= 0,
```

Comment:
How could the length be less than zero? You may forget the definition of length.